source/sigv4.c

```diff
@@ -2434,7 +2481,8 @@ static int32_t hmacFinal( HmacContext_t * pHmacContext,
          * this is by performing XOR on each byte of the inner-padded key (ipad ^ opad).  */
         for( i = 0U; i < pCryptoInterface->hashBlockLen; i++ )
         {
-            pHmacContext->key[ i ] ^= ( HMAC_INNER_PAD_BYTE ^ HMAC_OUTER_PAD_BYTE );
+            static const uint8_t padByteForOuterKey = HMAC_INNER_PAD_BYTE ^ HMAC_OUTER_PAD_BYTE;
```

Comment:
Is adding a new const and relying on the compiler to optimize it for you better than just defining a macro that's 0x6A?

Reply:
I stylistically prefer storing only publicly known information about the algorithm/protocol in the collection of macro constants in the header file. 
Wouldn't use of `static const` guarantee that the value is computed only at static initialization?

Reply:
Just thought it looked odd seeing a constant here instead of a macro. I'm not sure what's so different about 0x6A. It's just the XOR of 0x36 and 0x5C, the two constants defined in the HMAC RFC.

Reply:
Okay, I have added a macro constant for consistency with use of macros for constants